airbyte-cdk/bulk/toolkits/load-db/src/main/kotlin/io/airbyte/cdk/load/orchestration/legacy_typing_deduping/TypingDedupingRawTableOperations.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.load.orchestration.legacy_typing_deduping
+
+import io.airbyte.cdk.load.orchestration.TableName
+
+interface TypingDedupingRawTableOperations {
```

Comment:
this is the other half of StorageOperation, which deals with raw tables. It's an interface b/c this is super destination-specific.